l1-contracts/src/core/libraries/ConstantsGen.sol

```diff
@@ -47,8 +47,6 @@ library Constants {
   uint256 internal constant MAX_NOTE_ENCRYPTED_LOGS_PER_TX = 64;
   uint256 internal constant MAX_ENCRYPTED_LOGS_PER_TX = 8;
   uint256 internal constant MAX_UNENCRYPTED_LOGS_PER_TX = 8;
-  uint256 internal constant NUM_ENCRYPTED_LOGS_HASHES_PER_TX = 1;
-  uint256 internal constant NUM_UNENCRYPTED_LOGS_HASHES_PER_TX = 1;
```

Comment:
I nuked these 2 constants because they didn't really made much sense - there would always be one hash for each log type. Also a new constant was not introduced here for note logs which shows how unnecessary it is.